src/api/models/index.js

```diff
@@ -10,7 +10,8 @@ const models = {
   User: sequelize.import('./user'),
   Article: sequelize.import('./article'),
   Category: sequelize.import('./category'),
-  Rating: sequelize.import('./rating')
+  Rating: sequelize.import('./rating'),
+  TermsAndConditions: sequelize.import('./termsAndConditions.js')
```

Comment:
can you remove the .js? to have a unique structure

Reply:
alright

Reply:
done